examples/sparse_tensor.cu

```diff
@@ -133,10 +133,38 @@ int main([[maybe_unused]] int argc, [[maybe_unused]] char **argv)
   print(C);
 
   //
-  // Verify by computing the equivelent dense GEMM.
+  // Verify by computing the equivalent dense GEMM.
   //
   (C = matmul(A, B)).run(exec);
   print(C);
 
+  //
+  // Creates a CSR matrix which is used to solve the following
+  // system of equations AX=Y, where X is the unknown.
+  //
+  // | 1 2 0 0 |   | 1 5 |   |  5 17 |
+  // | 0 3 0 0 | x | 2 6 | = |  6 18 |
+  // | 0 0 4 0 |   | 3 7 |   | 12 28 |
+  // | 0 0 0 5 |   | 4 8 |   | 20 40 |
+  //
+  // Note that X and Y are presented by reshaping a 1-dim
+  // representation of the column-major storage, because the
+  // underlying library currently only supports column-major.
+  //
+  tensor_t<float, 1> coeffs{{nse}};
```

Comment:
`make_tensor` should be used instead of a `tensor_t` directly in all these cases

Reply:
changed all examples